gcsfs/core.py

```diff
@@ -488,6 +491,25 @@ def _call(self, method, path, *args, **kwargs):
             out = r.content
         return out
 
+    @_tracemethod
+    def _post(self, path, **kwargs):
```

Comment:
Can this not be achieved via `_call(method='post')`?

Reply:
Not directly as `_call()` returns `response.json` whereas most `post` requests need to complete response to read the headers and/or json body

I am not familiar enough with `requests` to know how to combine.

Reply:
Fair enough, I suppose they are fundamentally different